### ISSUE TITLE
Error on QuickStart examples page on stackstorm website

### ISSUE THREAD

**Issue by maltieriClarity** [StackStorm/st2docs#298]
Leaving this here per a request to do so when I mentioned it on IRC:

on the stackstorms docs pages (start page [start.html]) ... the example:

```
#And for fun, same post with st2" is broken -- "# And for fun, same post with st2
# st2 run core.http method=POST body='{"you": "too", "name": "st2"}' url=https://localhost/api/v1/webhooks/sample headers='x-auth-token=put_token_here;content-type=application/json' verify_ssl_cert=False"
```

`;` should be a `,`

**Comment by LindsayHill** [comment]
Looks like this got truncated @maltieriClarity? What was the rest of this, after "should be a" ?

**Comment by LindsayHill** [comment]
Addressed by #349 doc update
